tests/ServerListener.php

```diff
@@ -0,0 +1,40 @@
+<?php
```

Comment:
Should be in `src/` to be usable by the consumers, can you also use it in the project itself? It may speed out builds.

Reply:
Done!